pages/table/resizable-columns.page.tsx

```diff
@@ -173,6 +175,15 @@ export default function App() {
             <Checkbox checked={withSelection} onChange={event => setUrlParams({ withSelection: event.detail.checked })}>
               With row selection
             </Checkbox>
+            <Checkbox
+              checked={enableKeyboardNavigation}
+              onChange={event => {
+                setUrlParams({ enableKeyboardNavigation: event.detail.checked });
```

Comment:
Enabling keyboard navigation makes all row cells focusable with arrow keys but not header cells (unless they have interactive elements in them). Is this intentional?

Reply:
I see that if resizable columns are enabled, then the resizers are focusable and the header cells themselves are not (again, unless they have interactive elements). Just wanted to confirm if this a conscious decision?

Reply:
Yes, it is. When the column is not sortable but resizable the resize handle is the only focusable element of that column and that's why it is chosen as the navigation target. Likewise, if a body cell would include a text content and a button - only the button but not the text content nor the entire cell will be user-focusable.